lib/LaTeXML/resources/XSLT/LaTeXML-block-xhtml.xsl

```diff
@@ -154,6 +154,9 @@
         <xsl:call-template name="add_data_attribute">
           <xsl:with-param name="name" select="'href'"/>
         </xsl:call-template>
+        <xsl:if test="$USE_HTML5='true'">
```

Comment:
But data links work in xhtml generally, just (apparently) not in epub; do we need a separate switch for epub?

Reply:
The reasoning behind the `USE_HTML5` test is because the `download` attribute was not valid pre-HTML5 – it does not appear in the XHTML1.0/1.1 DTDs. An abundance of precaution, maybe?

For EPUB, I was going to submit a PR overriding `match="ltx:listing[@data]"` with an empty template in the EPUB3 stylesheet, rather than adding a flag in the XHTML template. Does that sound reasonable?

Reply:
@brucemiller the EPUB3 fix was actually a one-liner #1464, I could have sent it right away. With the two PRs, the end result is: all plain HTML outputs have the old download button; HTML5 and "XHTML5" get the additional `download` attribute; EPUB3 has no button at all (at least until there is a way of including the code as an attachment).